src/qt/test/test_main.cpp

```diff
@@ -54,6 +54,7 @@ int main(int argc, char* argv[])
 
     NodeContext node_context;
     std::unique_ptr<interfaces::Node> node = interfaces::MakeNode(&node_context);
+    gArgs.ForceSetArg("-listen", "0");
```

Comment:
Same for listenonion?

        f.write("listenonion=0\n")
        f.write("discover=0\n")
        f.write("dnsseed=0\n")
        f.write("fixedseeds=0\n")
        f.write("upnp=0\n")
        f.write("natpmp=0\n")

Reply:
Added those overrides as well.